sed/config/default.yaml

```diff
@@ -2,6 +2,14 @@ core:
   # The loader to use. The generic loader allows for loading various files types directly using dask.
   loader: generic
 
+  # If loader is set to flash, the following parameters are necessary
+  # Needs to be user defined
+  beamtime_id: None
+  year: None
```

Comment:
You cannot provide None objects in yaml. They end up as "None" strings. In addition, it also does not make any sense to provide None defaults. You anyways catch the case that any of these entries is not present, and then return a meaningful error message. So just leave them away, and have the user define them for a given experiment.

Reply:
Nit pick: That's not entirely true. You can use `None` in yaml files if you add `!!python/none`, e.g.
```E0: !!python/none None```
That's perfectly valid yaml. If that's make sense or not is a different story.

Reply:
You are right about it. But I just added it here for case of testing. But perhaps creating a dict within the test context is better.